docs/dc-apps-performance-toolkit-user-guide-jira.md

```diff
@@ -405,25 +406,21 @@ Review `results_summary.log` file under artifacts dir location. Make sure that o
 
 #### <a id="regressionrun2"></a> Run 2 (~50 min + Lucene Index timing test)
 
-If you are submitting a Jira app, you are required to conduct a Lucene Index timing test. This involves conducting a foreground re-index on a single-node Data Center deployment (without and with your app installed) and a dataset that has 1M issues.
-
-First, benchmark your re-index time without your app installed:
+If you are submitting a Jira app, you are required to conduct a Lucene Index timing test. This involves conducting a foreground re-index on a single-node Data Center deployment (with your app installed) and a dataset that has 1M issues.
 
 {{% note %}}
-Jira 7 index time for 1M issues on a User Guide [recommended configuration](#quick-start-parameters) is about ~100 min, Jira 8 index time is about ~40 min.
+Jira 7 index time for 1M issues on a User Guide [recommended configuration](#quick-start-parameters) is about ~100 min, Jira 8 index time is about ~30 min.
 {{% /note %}}
 
-1. Go to **![cog icon](/platform/marketplace/images/cog.png) &gt; System &gt; Indexing**.
-1. Select the **Lock one Jira node and rebuild index** option.
-1. Click **Re-Index** and wait until re-indexing is completed.
-1. **Take a screenshot of the acknowledgment screen** displaying the re-index time and Lucene index timing.
-1. Attach the screenshot to your DC HELP ticket.
+{{% note %}}
+If your Amazon RDS DB instance classes less then db.m5.xlarge you need wait (~2 hours after first reindex) before starting reindex.
```

Comment:
If your Amazon RDS DB instance class is lower then `db.m5.xlarge` it is required to wait ~2 hours after previous reindex finish before starting a new one.